internal/resolver/mtaresolver_test.go

```diff
@@ -106,45 +102,45 @@ var _ = Describe("Resolve", func() {
 		wd := getTestPath("test-project")
 		yamlPath := getTestPath("test-project", "mta.yaml")
 		envGetter = mockEnvGetterWithVcapServices
-		callResolveAndValidateOutput(wd, "eb-java", yamlPath, expected)
+		callResolveAndValidateOutput(wd, "eb-java", yamlPath, expected, "")
 	})
 	It("Sanity - working dir not provided", func() {
 		yamlPath := getTestPath("test-project", "mta.yaml")
 		envGetter = mockEnvGetterExtWithVcapServices
-		callResolveAndValidateOutput("", "eb-java", yamlPath, expected)
+		callResolveAndValidateOutput("", "eb-java", yamlPath, expected, "")
 
 	})
 	It("Sanity - environment file name different from the default name (.env)", func() {
 		wd := getTestPath("test-project2")
-		yamlPath := getTestPath("test-project2", "mta.yaml")
+		yamlPath := getTestPath("test-project", "mta.yaml")
 		envGetter = mockEnvGetterWithVcapServices
 		callResolveAndValidateOutput(wd, "eb-java", yamlPath, expected, ".env2")
```

Comment:
If the `.env2` file is the same as the `.env` file you can't know in this test that the file name is taken into account. Please adjust the test - change some values in `.env2` and send the `expected` accordingly (with the new values). Of course, it should be a different variable than the original `expected` so it won't affect other tests.

Reply:
Done